packages/open-next/src/adapters/cache.ts

```diff
@@ -49,22 +54,24 @@ export default class Cache {
 
     const softTags = typeof options === "object" ? options.softTags : [];
     const tags = typeof options === "object" ? options.tags : [];
-    return isFetchCache(options)
-      ? this.getFetchCache(key, softTags, tags)
-      : this.getIncrementalCache(key);
+    const isDataCache = isFetchCache(options);
+    return isDataCache
```

Comment:
nit: I liked the former version better (data/fetch/data sounds weird)

```suggestion
    return isFetchCache(options)
```